fem/transfer.hpp

```diff
@@ -305,13 +305,130 @@ class L2ProjectionGridTransfer : public GridTransfer
       virtual void ProlongateTranspose(const Vector& x, Vector& y) const;
       virtual void SetRelTol(double p_rtol_);
       virtual void SetAbsTol(double p_atol_);
+   protected:
+      /// Sets default shared options for the CGSolver
+      void InitializeCGSolver();
```

Comment:
```suggestion
      /// Sets default shared options for the CGSolver
      void SetDefaultCGOptions();
```

Maybe add a remark in the comment about what settings are actually set.